### DISCUSSION TITLE
Led config in the web interface

### DISCUSSION THREAD

**Discussion by poaudet** [ESPresense/ESPresense#130] [Ideas]
I've simply commented the line in the code to turn off the LED that blink.
I'm not really good at coding, so I propose this here. Maybe I'll give it a try to send a pull request, but if anyone want to add this feature: Add a option in the web interface to turn off the activity LED.

**Comment by B-Hartley** [comment]
I'd like that too. 
I haven't even looked at how to tweak the code.
